boost-gradle/src/main/groovy/io/openliberty/boost/gradle/tasks/BoostPackageTask.groovy

```diff
@@ -139,15 +142,15 @@ public class BoostPackageTask extends AbstractBoostTask {
 
     protected List<String> getSpringBootDependencies() {
 
-        List<String> springBootStarters = new ArrayList<String>()
+        List<String> springBootDependencies = new ArrayList<String>()
 
-        project.configurations.compile.dependencies.each { Dependency art ->
-            if (art.getGroup().equals("org.springframework")) {
-                springBootStarters.add(art.getName())
+        project.configurations.compile.resolvedConfiguration.resolvedArtifacts.each { art ->
+            if ("${art.name}".contains("spring")) {
```

Comment:
This is an external.   I know our doc is lagging, but this would be in complete doc, right?   

Since this PR is otherwise pretty complete, could we take this as a cue to start adding this as doc somehow?    

How? ...let's discuss more.

Reply:
Hmm... after mulling it over, I realized all this code does is filter or funnel a set of dependencies into `io.openliberty.boost.common.utils.SpringBootUtil.getLibertyFeaturesForSpringBoot(String, List<String>, BoostLoggerI)`.   It's the `io.openliberty.boost.gradle.utils.GradleProjectUtil.findSpringBootVersion(Project)` which detects it as a SpringBoot project.  

So ... we can still talk about doc, but my comment can be ignored.